src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -85,6 +85,11 @@ private Command(CommandSpec commandSpec)
 
             return command;
         }
+
+        public static Command Create(CommandSpec commandSpec)
```

Comment:
If we need to expose this, we may as well make the constructor public. The original intent of the factory pattern here was to capture the logic for creating particular kinds of commands in one place...